apps/connect/pyproject.toml

```diff
@@ -40,8 +40,8 @@ pyyaml = "^6.0.1"
 
 #ftrack
 ftrack-action-handler = "*"
-ftrack-utils = {  version = "^2.0.0", source = "testpypi", allow-prereleases = true }
```

Comment:
Are we documenting this somewhere before we remove it (release candidates)?

Reply:
Perhaps quickly add this the the root README

Reply:
Not right now, we need to find/create a tool to be able to make it easier to pick from testpypi / allow pre-releases.